torch/_torch_docs.py

```diff
@@ -2701,6 +2701,43 @@ def merge_dicts(*dicts):
     input (Tensor): the PyTorch tensor to test
 """)
 
+add_docstr(torch.is_nonzero,
+           r"""
+is_nonzero(input) -> (bool)
+
+Returns True if the :attr:`input` is a scalar which is not equal to zero
+after type conversions.
+i.e. not equal to ``torch.Tensor([0.])`` or ``torch.Tensor([0])`` or
+``torch.Tensor([False])``.
+Throws a ``RuntimeError`` if ``torch.numel() != 1`` (after coalescing in case
+of sparse tensors).
+
+Args:
+    input (Tensor): the PyTorch tensor to test
+Example::
+
+    >>> torch.is_nonzero(torch.Tensor([0.]))
+    False
+    >>> torch.is_nonzero(torch.Tensor([1.5]))
+    True
+    >>> torch.is_nonzero(torch.Tensor([False]))
+    False
+    >>> torch.is_nonzero(torch.Tensor([True]))
+    True
+    >>> torch.is_nonzero(torch.Tensor([0]))
+    False
+    >>> torch.is_nonzero(torch.Tensor([3]))
+    True
+    >>> torch.is_nonzero(torch.Tensor([]))
```

Comment:
Instead of a tensor with no values throwing a runtime error, how about a tensor with a few values? Like torch.tensor((1, 3, 5))?

Reply:
Updated, still retaining empty example for clarification